hooks/useVoteDirections.ts

```diff
@@ -0,0 +1,33 @@
+import { useMemo } from 'react';
+import { Vote } from '../types/Vote';
+
+const sortOnLikes = (a: Vote, b: Vote) => {
```

Comment:
nice thanks, people have been asking for a time sort option on the prop view so this might make that easier